src/Form/icons/IconRadioButton.js

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames/dedupe';
+import React from 'react';
+
+class IconRadioButton extends React.Component {
+  render() {
+    let {checked, className, shadowClassName, smallCircleClassName} = this.props;
+    let iconClasses = classNames('icon-radio-button', className);
+    let shadowClasses = classNames(
+      'shadow',
```

Comment:
Can we change this to `icon-radio-button-shadow`? I think `shadow` will be too generic.